docs/security/trust/content_trust.md

```diff
@@ -152,14 +152,36 @@ In an environment where `DOCKER_CONTENT_TRUST` is set, you can use the
 `--disable-content-trust` flag to run individual operations on tagged images
 without content trust on an as-needed basis.
 
-```bash
-$  docker pull --disable-content-trust docker/trusttest:untrusted
+Consider the following Dockerfile that uses an untrusted base image:
+
+```
+$  cat Dockerfile
+FROM docker/trusttest:latest
+RUN echo
+```
+
+In order to build a container successfully using this Dockerfile, one can do:
+
+```
+$  docker build --disable-content-trust -t <username>/nottrusttest:testing .
```

Comment:
@mansinahar Sounds reasonable to me!  Thank you for clarifying!  I'm also wondering whether names like "nottrusted" and "maybetrusted:signed" would make it easier to determine what the intent is with the image?